x-pack/plugins/ml/public/application/trained_models/models_management/models_list.tsx

```diff
@@ -227,6 +234,39 @@ export const ModelsList: FC = () => {
     }
   }, []);
 
+  /**
+   * Updates model items with deployment stats;
+   *
+   * We have to fetch all deployment stats on each update,
+   * because for stopped models the API returns 404 response.
+   */
+  const fetchDeploymentStats = useCallback(async (modelItems: ModelItem[]) => {
```

Comment:
as this is mutating the `modelItems` maybe it should be called something like `populateDeploymentStats`

Reply:
renamed in 4665cf721de51133b3d6b28006cec0ebd8b81c1a